1608.special-array-with-x-elements-greater-than-or-equal-x.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/special-array-with-x-elements-greater-than-or-equal-x/
+ */
+
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var specialArray = function(nums) {
+    for (let x = nums.length; x >= 0; x--) {
+        let counter = 0;
+        
+        for (let i = 0; i < nums.length; i++) {
+            if (nums[i] >= x) counter++;
+        }
+        
+        if (x === counter) return x;
+        counter = 0;
```

Comment:
You can remove this line because `counter` is initialized to 0 at the beginning of each loop.